sos/report/plugins/microshift.py

```diff
@@ -146,7 +147,13 @@ def setup(self):
         Output format for this function is based on `oc adm inspect` command,
         which is used to retrieve all API resources from the cluster.
         """
-        self.add_forbidden_path('/var/lib/microshift')
+        self.add_service_status('greenboot-healthcheck')
+        self.add_journal('greenboot-healthcheck')
```

Comment:
If you add this service to the `services` tuple (akin to `packages`, `commands`, etc...), these collections become implicit.

Reply:
Done.